Docs/source/FlowChart.rst

```diff
@@ -312,7 +312,7 @@ here, consistent with the names used in the code:
 - ``new_source`` is a MultiFab reference to the new-time-level ``Source_Type`` data.
 
 
-Single Step Flowchat
+Single Step Flowchart
 --------------------
```

Comment:
can you extend the `-` by one more character too?  The `-----` line must be the same length as the heading in Sphinx.

Reply:
Done.